src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,10 +35,16 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
+  //private final DriveSystem driveSystem;
+  private final LiftSystem lSystem;
+
+  private JoystickButton liftToButton;
+  private JoystickButton liftSpeedButton;
```

Comment:
are these used anymore? if not they can be deleted

if so, button declarations should be after controller rather than subsystem